tests/conftest.py

```diff
@@ -138,3 +140,22 @@ def backend(request):
 )
 def interpcode(request):
     yield request.param
+
+
+@pytest.fixture(scope='function')
+def datadir(tmpdir, request):
+    '''
+    Fixture responsible for searching a folder with the same name of test
+    module and, if available, moving all contents to a temporary directory so
+    tests can use them freely.
+    '''
+    # this gets the module name (e.g. /path/to/pyhf/tests/test_schema.py)
+    # and then gets the directory by removing the suffix (e.g. /path/to/pyhf/tests/test_schema)
+    test_dir = pathlib.Path(request.module.__file__).with_suffix('')
+
+    if test_dir.is_dir():
+        distutils.dir_util.copy_tree(test_dir, tmpdir.strpath)
+        # shutil is nicer, but doesn't work: https://bugs.python.org/issue20849
+        # shutil.copytree(test_dir, tmpdir)
```

Comment:
Maybe I'm missing something, but why doesn't the linked PR fix this?

Reply:
I believe it's only in 3.8. My version of 3.7.3 has this error:

```
        if test_dir.is_dir():
>           shutil.copytree(test_dir, tmpdir, dirs_exist_ok=True)
E           TypeError: copytree() got an unexpected keyword argument 'dirs_exist_ok'
```

Reply:
Ah, sorry. You're correct. It only shows up in the `3.8` version of the docs: https://docs.python.org/3.8/library/shutil.html Thanks!